docs/customization/template.rst

```diff
@@ -98,6 +113,77 @@ Done! If you do not see any changes on the ``/admin/countries/new`` url, clear y
 
     $ php bin/console cache:clear
 
+How to customize templates via events?
+--------------------------------------
+
+Sylius uses the Events mechanism provided by the `SonataBlockBundle <https://sonata-project.org/bundles/block/master/doc/reference/events.html>`_.
+
+How to locate template events?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The events naming convention uses the routing to the place where we are adding it, but instead of ``_`` we are using ``.``,
+followed by a slot name (like ``sylius_admin_customer_show`` route results in the ``sylius.admin.customer.show.slot_name`` event).
+The slot name describes where exactly in the template's structure should the event occur.
+
+Let's see how the event is rendered in a default Sylius Admin template.
```

Comment:
This sentence sounds like it should be emphasized in some way.
Either that or it's not needed as the next line conveys pretty much the same info but w/ a *how to* example.